design/noobaa/system-read.md

```diff
@@ -0,0 +1,67 @@
+♜ [Rook NooBaa Design](README.md) /
+# System Read
+
+The operator will set the status of the NooBaa system to represent the current state of reconciling to the desired state.\
+Here is the example status structure as would be returned by a `kubectl get noobaa <system-name> -o yaml`:
+
+```yaml
+apiVersion: noobaa.rook.io/v1alpha1
+kind: System
+metadata:
+  name: noobaa-1
+  namespace: rook-noobaa
+spec:
+    # ...
+status:
+
+  health:
+    accounts: OK
```

Comment:
What do these look like if they are not healthy? Does that mean noobaa cannot connect to the backing store? For example, the creds might be incorrect?

Reply:
Ok, I see an example below.

Reply:
Thanks, resolving.